lib/loader.js

```diff
@@ -53,7 +53,7 @@ function createLoader() {
         //if parent id is a consumer, i.e. was registered with a prefix,
         //strip the prefix off since that's not part of the module name
         parentId = parentId.indexOf('.') > -1 ? parentId.substring(parentId.indexOf('.') + 1) : parentId;
-
+        
```

Comment:
whitespace (I can't recall if this project has the requirement to strip trailing whitespace ... I may just be used to seeing that in other code)

Reply:
My editor added that, but lint passes so I guess it's ok?